omnigibson/object_states/particle_modifier.py

```diff
@@ -1078,17 +1079,20 @@ def _initialize(self):
             self.projection_source_sphere.initialize()
             self.projection_source_sphere.visible = False
             # Rotate by 90 degrees in y-axis so that the projection visualization aligns with the projection mesh
-            self.projection_source_sphere.set_local_pose(orientation=T.euler2quat([0, np.pi / 2, 0]))
+            self.projection_source_sphere.set_local_pose(
+                orientation=T.euler2quat(th.tensor([0, math.pi / 2, 0], dtype=th.float32))
```

Comment:
note for myself, are all functions from `utils.transform_utils` returning torch tensors now?